python/pip_install/pip_repository.bzl

```diff
@@ -367,6 +367,10 @@ def _pip_repository_bzlmod_impl(rctx):
     else:
         build_contents += _bzlmod_pkg_aliases(repo_name, bzl_packages)
 
+    macro_tmpl = "@@{name}//:{{}}_{{}}".format(name = rctx.attr.name)
```

Comment:
Would you mind putting a brief comment about why we use "@@"? When and why to use the apparent name vs canonical name isn't always clear and is a bit confusing.